services/121-service/test/helpers/utility.helper.ts

```diff
@@ -39,7 +39,7 @@ export async function getAccessToken(): Promise<string> {
 }
 
 export function loginAsProgramManager(): Promise<request.Response> {
-  return getServer().post(`/user/login`).send({
```

Comment:
I think this change will break our external api integrations (e.g. kobo connect/espocrm) @jannisvisser